collector/perf_schema_replication_applier_status_by_worker.go

```diff
@@ -0,0 +1,223 @@
+// Copyright 2018 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package collector
+
+import (
+	"context"
+	"database/sql"
+	"strconv"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const perfReplicationApplierStatsByWorkerQuery = `
+	SELECT CHANNEL_NAME, WORKER_ID, 
```

Comment:
To make this easier to read and avoid long lines, let's format it with  a column on each line. See [this example](https://github.com/prometheus/mysqld_exporter/blob/master/collector/perf_schema_table_lock_waits.go#L25-L50)